### DISCUSSION TITLE
Hires. fix maybe?

### DISCUSSION THREAD

**Discussion by williamsblood** [lllyasviel/Fooocus#761] [Ideas]
Could we consider implementing a high-resolution fix option (Hires. fix from automatic1111) for future use? I understand that upscaling is possible, but it doesn't enhance the image's quality as effectively.

**Comment by lllyasviel** [comment]
latent resampling does not work well in sdxl latent space.
lanzcos is worse than neural methods in most tests.
neural method like esrgan\swinir\ldsr is already in fooocus and is optimized for sdxl, called Upscale.

when user want to consider anything about image qualiy and you cannot find the option in Fooocus, it means Fooocus already use better method.
